content/news/playtest-20170826.md

```diff
@@ -0,0 +1,16 @@
+A month after the previous playtest, we're pleased to announce the new release candidate, playtest-20170826.
+
+In addition to all the new features and improvements the previous playtest brought along, in this new playtest we fixed over 30 issues that were identified through player and modder feedback. Among the more noticable ones were the movement 'stuttering' that ground units were showing, as well as some issues with the new command bar and some other UI hiccups.
+
+Furthermore, the Red Alert mod in particular received a larger balance overhaul, along with a new map (Agenda). 'Agenda' and the updated 'Great Sahara 2' now serve as examples of more prominent use of civilian 'tech' structures that provide various benefits when captured. Tiberian Dawn's balance received some further polishing as well, making bridges more robust (in particular to light weapons and flames) as well as making Orcas and rocket infantry less effective versus infantry in general.
+
+A new version of the Mod SDK is [also now available](https://github.com/OpenRA/OpenRAModSDK/releases/tag/20170826) with a few improvements and bug fixes.  The big new feature is the ability to [automatically create installers](https://github.com/OpenRA/OpenRAModSDK/wiki/Generating-Release-Installers) for your mod "in the cloud".  We encourage everyone who is working on mods for the last release to try out the ModSDK and report back any issues you find or suggestions that you may have, before it is too late!
```

Comment:
"before it is too late!" is perhaps a bit to ultimatum-ey.  This could be replaced with "so that we can iron out any remaining issues before the legacy oramod infrastructure is removed.".